doc/flame/inputs/tap_events.md

```diff
@@ -15,33 +15,24 @@ Multiple tap events can occur at the same time, especially if the user has multi
 cases will be handled correctly by Flame, and you can even keep track of the events by using their
 `pointerId` property.
 
-It takes only a few simple steps to enable these events for your game:
+For those components that you want to respond to taps, add the `TapCallbacks` mixin.
 
-1. Add the `HasTappableComponents` mixin to your main game class:
+- This mixin adds four overridable methods to your component: `onTapDown`, `onTapUp`,
+  `onTapCancel`, and `onLongTapDown`. By default, each of these methods does nothing, they need
+  to be overridden in order to perform any function.
+- In addition, the component must implement the `containsLocalPoint()` method -- this method
```

Comment:
Can we have a default implementation for `containsLocalPoint`? Some that would at least rely on the bounding rectangle of the component?

Reply:
Okay, reading down bellow, I see that this is the case for `PositionComponents`, then probably just change this to "unless your component is a `PositionComponent` the component must implement..."?

Reply:
Good idea, updated!